### DISCUSSION TITLE
Persistent MobiFlight variables

### DISCUSSION THREAD

**Discussion by tigert** [MobiFlight/MobiFlight-Connector#1649] [Ideas]
**Is your feature request related to a problem? Please describe.**
It is difficult to create persistent settings in MobiFlight, for example if you create an audio panel or a "multi-panel" where you can select which radio to display at a time. This works well for a selector switch that has a defined state, but if you create an audio panel with pushbuttons for example to select the microphone or radio you listen to, those settings will always get reset when MobiFlight is restarted, and you basically have to hardcode the "default" setting if you always use a specific layout. Many avionics however do remember their set states accross flights if you have a plane you fly frequently.

**Describe the solution you'd like**
Introduce a "remember set value between sessions" setting to MobiFlight variables. Sometimes you depend on the initial value being zero, so that should not be removed, but optionally allow persistent values that get saved on the user config with their config item ID hash for example.

**Describe alternatives you've considered**
Some planes naturally remember their state, but this is very plane specific and not an universal solution.

**Comment by DocMoebiuz** [comment]
The use case is not specific enough. This issue requires more discussion, I will convert it.
